jenkins/scripts/dynamic_worker_workflow/run_clean.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -eux
+
+if [[ "${IMAGE_OS}" == "ubuntu" ]]; then
+    export CONTAINER_RUNTIME="docker"
+    export EPHEMERAL_CLUSTER="kind"
+else
+    export EPHEMERAL_CLUSTER="minikube"
+fi
```

Comment:
Add empty line between conditions for readability.

Reply:
added